lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1506,14 +1506,11 @@ def _resource_sku_to_capability(  # noqa: C901
         ](is_allow_set=True, items=[])
         for sku_capability in resource_sku.capabilities:
             name = sku_capability.name
+            vcpus = 0
             if name == "vCPUsAvailable":
                 node_space.core_count = int(sku_capability.value)
```

Comment:
The code logic can be more clear. It can have two variables, to save both value, and then pick one of them by priority like this. It makes the code more readable, because the core_count is set in one place.
```python
        vcpus = 0            
        vcpus_available = 0
            if name == "vCPUsAvailable":
                vcpus_available = int(sku_capability.value)
            elif name == "vCPUs":
                vcpus = int(sku_capability.value)

            ...

        if vcpus_avaiable:
            node_space.core_count = vcpus_avaiable
        else:
            node_space.core_count = vcpus
```